ph5/clients/ph5tostationxml.py

```diff
@@ -417,21 +419,29 @@ def get_response_inv(self, obs_channel):
             self.manager.ph5.read_response_t()
             Response_t = \
                 self.manager.ph5.get_response_t_by_n_i(self.response_table_n_i)
-            response_file_das_a_name = Response_t.get('response_file_das_a',
-                                                      None)
-            response_file_sensor_a_name = Response_t.get(
-                                                    'response_file_sensor_a',
-                                                    None)
+            if Response_t:
+                response_file_das_a_name = Response_t.get(
+                    'response_file_das_a', None)
+                response_file_sensor_a_name = Response_t.get(
+                    'response_file_sensor_a', None)
+            else:
+                LOGGER.error('Response table not found')
+                return Response()
             # parse datalogger response
             if response_file_das_a_name:
                 response_file_das_a = \
                     self.manager.ph5.ph5_g_responses.get_response(
                                                     response_file_das_a_name
                                             )
+
                 with io.BytesIO(response_file_das_a) as buf:
                     buf.seek(0, 0)
-                    dl_resp = read_inventory(buf, format="RESP")
-                dl_resp = dl_resp[0][0][0].response
+                    if _is_resp(buf):
```

Comment:
This use of a private ObsPy method seems dangerous as it could change without notice. It also moves the `buf` file pointer so `buf.seek(0, 0)` needs to be called again after `_is_resp(buf)`.

Reply:
we can write our own method for checking if it is RESP file. Should just be finding the first non comment line and chacking it against a regular expression